compiler/rustc_codegen_llvm/src/back/lto.rs

```diff
@@ -528,31 +528,37 @@ fn thin_lto(
     }
 }
 
-fn enable_autodiff_settings(ad: &[config::AutoDiff]) {
+#[cfg(feature = "llvm_enzyme")]
+pub(crate) fn enable_autodiff_settings(
+    sysroot: &rustc_session::config::Sysroot,
+    ad: &[config::AutoDiff],
+) {
+    let mut enzyme = llvm::EnzymeWrapper::get_or_init(sysroot);
```

Comment:
This can use `get_instance`, right? The `get_or_init` call inside `LlvmCodegenBackend` should already be called at this point.